src/js/modules/infragistics.ui.editors.js

```diff
@@ -1007,6 +1007,11 @@
 			}
 		},
 		_setFocus: function (event) {
+			if (this._shouldNotFocusInput) {

```

Comment:
If this flag applies only to the date picker it probably shouldn't be in the base `_setFocus`